futurecommander_gui/app_node/filesystem/client.js

```diff
@@ -23,6 +23,9 @@ const { Request } = require('./request');
 
 class FileSystemClient {
     constructor() {
+        console.log('Create FileSystemClient');
```

Comment:
Does this log will be read by someone?

Reply:
Whose wondering why that goddam filesystem ain't working i guess ...
Just kidding it's only me. I'll remove it.